### PR TITLE
README.md - Izuma changes

### PR DESCRIPTION
**Pull request by JanneKiiskila**
## Description

Fix email address & one ref to Pelion.

## Test instructions

Proof read.

### API change(s)

 - [x] Not applicable.
 - [ ] API is backwards compatible.
 - [ ] API documentation is updated.

### Example applications updated

 - [x] Not applicable.
 
 ### Changelog
 
 - [-] `CHANGELOG.md` updated. 
     - Not applicable.

### CODE REVIEW

**Review by petedyerarm — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This document contains the instructions for using and developing Edge.
 
-The full Edge documentation is [part of our Device Management documentation site](https://developer.pelion.com/docs/device-management-edge/latest/introduction/index.html), where you can also find the [API documentation](https://developer.pelion.com/docs/device-management-edge/latest/edge-api-references/index.html). For comments or questions about the documentation, please [email us](mailto:support@mbed.org).
+The full Edge documentation is [part of our Device Management documentation site](https://developer.pelion.com/docs/device-management-edge/latest/introduction/index.html), where you can also find the [API documentation](https://developer.pelion.com/docs/device-management-edge/latest/edge-api-references/index.html). For comments or questions about the documentation, please [email us](mailto:support@izumanetworks.com).
```
Comment (jenia81) [inline README.md:5]: The links that are currently Pelion will be fixed later on?

Comment (JanneKiiskila) [reply README.md:5]: Yes. Lots of links fixing once we get all the web stuff set up.

**Comment by JanneKiiskila** [comment]
https://github.com/PelionIoT/mbed-edge/pull/86 -> supercedes this one.

**Pull request closed without merging**